content/en/docs/js/exporters.md

```diff
@@ -107,5 +107,20 @@ const meter = new MeterProvider({
 
 ## OpenTelemetry Collector
 
-If you are looking for a vendor-agnostic way to receive, process and export your
-telemetry data follow the instructions to setup a [collector](/docs/collector/)
+To send trace data to a Collector you'll want to use the `exporter-collector` package:
+
+```shell
+npm install --save @opentelemetry/exporter-collector
+```
+
+And configure the exporter to point at your Collector endpoint:
+
+```javascript
+import { CollectorTraceExporter } from '@opentelemetry/exporter-collector';
```

Comment:
hmm, I guess this should instead be one of

`@opentelemetry/exporter-trace-otlp-[http/grpc/proto]`